lib/resources/Accounts.js

```diff
@@ -14,6 +14,7 @@ module.exports = StripeResource.extend({
   list: stripeMethod({
     method: 'GET',
     path: 'accounts',
+    autoPageable: true,
```

Comment:
So the idea is that we still need to add this parameter to a few dozen other list calls elsewhere right?

The `autoPageable` parameter seems a little unusual to me given that (1) only lists are pageable, and (2) all lists should be pageable. What do you think about naming this something like `methodType: 'list'` or something along those lines instead?

Reply:
Yeah, looks like I missed methods like `listTransactions`, etc – looks like there's 13 of those. 

I like `methodType: 'list'` a little better too, let's go with that. My thinking was "what if there's a list that isn't pageable" but that's pretty silly.

Reply:
✅